### PR TITLE
[ja] Translate /docs/contribute/style/hugo-shortcodes/ into Japanese

### LINKED ISSUE #32653: [ja] Translate /docs/contribute/style/hugo-shortcodes/ into Japanese
**This is a Feature Request**

**What would you like to be added**
A /docs/contribute/style/hugo-shortcodes/ page translated into Japanese

**Why is this needed**

The page has not been translated.

**Comments**

/assign

### PR DESCRIPTION
**Pull request by nasa9084**
Fix #32653

### CODE REVIEW

File: content/ja/docs/contribute/style/hugo-shortcodes/index.md
Comment (jihoon-seo) [inline content/ja/docs/contribute/style/hugo-shortcodes/index.md:54]: Japanese version exists.
(And there might be more links that can be updated to Japanese versions. I did not checked all.)

```suggestion
コンテンツを自動的に更新し、[用語集](/ja/docs/reference/glossary/)へのリンクを付与する挿入を使用して、用語を参照することができます。
```

**Review by jihoon-seo — COMMENTED** [review]
[Netlify build was successful](https://app.netlify.com/sites/kubernetes-io-main-staging/deploys/62448f69679623000967f69d), but its deployment was not.

Maybe pushing another commit will retrigger the Netlify build and deploy.

@nasa9084 PTAL!

**Comment by nasa9084** [comment]
It seems deployment has done well

**Review by riita10069 — APPROVED** [review]

File: content/ja/docs/contribute/style/hugo-shortcodes/index.md
Comment (s-kawamura-w664) [inline content/ja/docs/contribute/style/hugo-shortcodes/index.md:92]: When I looked this page on netlify's preview, the following was displayed.
```
例えば、 への参照方法は次の通りです
```

**Comment by nasa9084** [comment]
ah, thank you, honestly, I had not noticed.
let me hold this PR once...
/hold

**Comment by s-kawamura-w664** [comment]
@nasa9084
The api-reference shortcode issue has been fixed at #34272.
You can restart this PR.

**Comment by divya-mohan0209** [comment]
@nasa9084 : Please can you advise what exactly is it that is blocking the PR from a merge?

**Comment by nasa9084** [comment]
@divya-mohan0209 the blocker is @inductor is busy recently. only @inductor and I are approvers for Japanese content and I'm the author of this PR so need approval from @inductor...

**Comment by divya-mohan0209** [comment]
Hey @nasa9084 , this might merit a longer discussion during the subproject meeting but do y'all need help to onboard more contributors and promote them to approvers/reviewers? I ask because the PR has been lying dormant for ~5 months now waiting for a review from @inductor. I'm not trying to overstep or impose and understand that the cadence is generally slow in localizations, but from the leadership side of things, we would love to help in anyway we can. Please let us know if you need help.

**Comment by inductor** [comment]
Sorry I did not notice!

**Pull request merged**
